libs/auth/scopes/src/lib/application.scope.ts

```diff
@@ -0,0 +1,4 @@
+export enum ApplicationScope {
+  read = '@island.is/applications:read',
```

Comment:
Should we consider shortening the scopes' names? Is there a limit on the JWT token in IDS we should consider @valurefugl? I recall having problems with JWT size in Auth0 due to being overly verbose in claims.

Reply:
Yeah, good question. I did some testing a while back, and with our current production servers and proxies, we support up to 4KB header values. This allows us to use around 100 scopes of this length. It "should" be enough, but I'm definitely open to better naming conventions.

We want to namespaces to avoid conflicts between gov orgs. I like readability, but I know some OAuth2 providers use fairly obfuscated scope names. "island/app:r"?

Reply:
Am not aware of any limits in the IDS itself.